cmd/kops/get.go

```diff
@@ -138,6 +138,9 @@ func RunGet(context Factory, out io.Writer, options *GetOptions) error {
 	args := make([]string, 0)
 
 	clusters, err := buildClusters(args, clusterList)
+	if err != nil {
+		return fmt.Errorf("error on buildClusters(): %v", err)
```

Comment:
Just a note/nitpick

BTW really appreciate you fixing all of these!!!

In the future, if we can do human readable error messages, meaning "building the cluster" failed.